src/components/Admin/Admin.js

```diff
@@ -21,7 +21,7 @@ const WATCHLIST_TAB = 1;
 const LOCATIONS_TAB = 2;
 const CUSTOM_EVENTS_TAB = 3;
 const FB_PAGES_TAB = 4;
-const TRUSTED_TWITTER_ACCTS = 5;
+const TRUSTED_SOURCES = 5;
 const TWITTER_ACCOUNTS_TAB = 6;
```

Comment:
Please remove any remnants of Facebook and/or Twitter across all admin tabs.

Reply:
ok will do